housekeeper/store/api/find.py

```diff
@@ -53,20 +63,18 @@ def version(
         """Fetch a version from the store."""
         if version_id:
```

Comment:
I don't think this pattern makes sense - to pass multiple parameters and use if-statements. We should try to split these into separate smaller functions in general, it makes the code easier to test and understand.

Reply:
Saving the refactoring to a separate PR